packages/client/src/stores/state/state.store.ts

```diff
@@ -513,6 +513,18 @@ export class StateStore {
                     : null,
             );
 
+            const stringValue = value as string;
+            try {
+                if (stringValue !== undefined) {
+                    let jsonValue = JSON.parse(stringValue);
+                    if (jsonValue !== undefined)
+                        value = JSON.parse(stringValue);
+                }
+            } catch (e) {
+                console.log('error occurred while parsing');
+                return value;
+            }
+
```

Comment:
We shouldn't make modification to this file. What is the purpose of this?

Reply:
While linking variables to chart, at Echarts entire json came up as a string. So added the above fix. Now i removed the above fix

Reply:
While getting the variable value, it is coming as string. to apply the json string to echart, I parsed it and applied the value. I verified this changes once with vega chart also, at that time both chart worked with the above changes. Now I am removing this changes completely.